Detector/DetComponents/src/MergeLayers.cpp

```diff
@@ -55,16 +55,6 @@ StatusCode MergeLayers::initialize() {
             << endmsg;
     return StatusCode::FAILURE;
   }
-  // check sizes of new volumes in the list - it must sum to the total number of volumes
```

Comment:
Why did you remove this check?

Reply:
It was causing troubles if the same volume name is used in different systems. We have "layer" in the ECAL and the same name is used also in some other subdetector. After discussion with Anna, the author of this tool, we decided to remove this check and leave the responsibility on the user of the code.